quupe-RN/js/redux/modules/Token.js

```diff
@@ -6,6 +6,7 @@ const GET_ERROR = 'GET_ERROR';
 const DELETE_TOKEN = 'DELETE_TOKEN';
 const LOGIN_CANCELLED = 'LOGIN_CANCELLED';
 const LOGIN_ERROR = 'LOGIN_ERROR';
+const ACCEPT_TERMS = 'ACCEPT_TERMS';
```

Comment:
I think this should be saved in the database. Through the current approach, If the app closes completely the user may have to accept the terms again.